Lib/email/generator.py

```diff
@@ -392,7 +392,7 @@ def _make_boundary(cls, text=None):
     def _compile_re(cls, s, flags):
         return re.compile(s, flags)
 
-
+
```

Comment:
(This blank line should stay, or there'll only be one blank line in between the class definition and the function above it)